libqfieldsync/offline_converter.py

```diff
@@ -266,15 +268,15 @@ def _convert(self, project: QgsProject) -> None:
             elif (
                 layer_action == SyncAction.COPY or layer_action == SyncAction.NO_ACTION
             ):
-                copied_files = layer_source.copy(self.export_folder, copied_files)
+                copied_files = layer_source.copy(
+                    self._export_filename.parent, copied_files
```

Comment:
```suggestion
                    self._export_filename.parent,
                    copied_files,
```

It is all parameters to be on separate lines when they cannot fit on the same line as the parantesis.